qcut/docs/task/qwen-dashscope/qwen-chat.py

```
@@ -0,0 +1,41 @@
+    )
+
+    try:
+        completion = client.chat.completions.create(
+            model="qwen-plus",
+            messages=[
+                {"role": "system", "content": "You are a helpful assistant."},
+                {"role": "user", "content": "Who are you?"},
+            ],
+        )
+        print(completion.choices[0].message.content)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)